srcpkgs/gcalcli/template

```diff
@@ -0,0 +1,19 @@
+# Template file for 'gcalcli'
+pkgname=gcalcli
+version=4.3.0
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-setuptools python3-wheel"
+depends="python3-dateutil python3-google-api-python-client
+	python3-httplib2 python3-oauth2client python3-parsedatetime
+	python3-vobject" #This last one is optional.
```

Comment:
```suggestion
 python3-httplib2 python3-oauth2client python3-parsedatetime
 python3-vobject"
```

Reply:
If I commit your suggestions, it will create a second commit, right? Perhaps I could modify it locally and force push again? I remember messing things horribly last time I did this.

Reply:
yeah you don't have to use the commit suggestion button, it's just handy when reviewing